components/form/builder/src/reducer/operators.js

```diff
@@ -104,5 +105,9 @@ export const operators = {
     const isEmpty = !fieldValue.length
     const hasNotExpectedValues = values.some(i => !fieldValue.includes(i))
     return isEmpty || hasNotExpectedValues
+  },
+  [HIDDEN]: (id, fields) => {
+    const {hidden} = pickFieldById(fields, id)
+    return hidden === true
```

Comment:
Do you prefer to return hidden directly?

Reply:
yes

Reply:
If you want to make sure it is a boolean you can cast it using `Boolean(hidden)` or `!!hidden`

Reply:
⬆️ agree